src/Qmmands/Mapping/CommandMappingException.cs

```diff
@@ -13,7 +13,7 @@ public sealed class CommandMappingException : Exception
         public Command Command { get; }
 
         /// <summary>
-        ///     Gets the segment to map this exception occurred for.
+        ///     Gets the segment of the map this exception occurred at.
```

Comment:
The segment might not necessarily have been mapped before. Would "Gets the segment to map <ins>that</ins> this exception occurred for." make more sense?